andela/vidar-ah-frontend
# #164139642 Implement user registration on the app.

#### What does this PR do?
It consumes the user registration endpoint using react and redux.

#### Description of Task to be completed?
Users should be able to signup into Authors Haven by providing a name, username, email and password. On successful registration, the user is required to verify their account using the link sent to their email.

#### How should this be manually tested?
- Visit the app URL `https://vidar-ah-frontend-staging.herokuapp.com/signup`
- Provide your registration details ( name, username, email and password).
- If the credentials are valid, you'll be successfully registered to the app
- If the credentials are invalid, you'll get error(s) messages.
- After registration, verify the account using verification link from the email.
- The link redirects to a page that verifies the user's email.

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/164139642

#### Screenshots (if appropriate)


#### Questions:
N/A
